t/react_template/public/index.html

```diff
@@ -9,6 +9,8 @@
       name="description"
       content="React template"
     />
+    <!-- sctrict to add trusted type when using dangerouslySetInnerHTML -->
```

Comment:
spelling? 
Suggest  <!-- content security policies to ..... -->, we'll be adding more in future.